masterlist.yaml

```diff
@@ -4212,10 +4212,11 @@ plugins:
 
   - name: 'Toccata as Elisif.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/12713/' ]
+    after: [ 'PAN_NPCs.esp' ]
```

Comment:
It should already do this, but there's no harm in having it added.

Reply:
ah, sorry! i remember having that custom rule for ages in LE as well because i don't think it always did